browser/brave_shields/ad_block_subscription_download_manager_getter.h

```diff
@@ -8,9 +8,7 @@
 
 #include "brave/components/brave_shields/browser/ad_block_subscription_download_manager.h"
 
-using brave_shields::AdBlockSubscriptionDownloadManager;
```

Comment:
don't use aliases in header files https://google.github.io/styleguide/cppguide.html#Aliases